packages/ui-components/src/components/calendar-day/calendar-day.types.ts

```diff
@@ -5,36 +5,25 @@ export interface ICalendarDay {
 	day: Number;
 	/** (optional) If `true`, the day is disabled */
 	disabled?: boolean;
-	/** (optional) If `true`, the day is with active style */
+	/** (optional) If `true`, the day is with active */
 	active?: boolean;
-	/** (optional) If `true`, the day is with 'in-range' style */
+	/** (optional) If `true`, the day is in range from a start date to the current hovered day */
```

Comment:
Is this only related to the "hover" state? Only by looking at this prop name, I assume this also has something to do with the "selected" state.